opencmiss/utils/zinc.py

```diff
@@ -380,6 +380,33 @@ def createImageField(fieldmodule, image_filename):
     return image_field
 
 
+def createVolumeImageField(fieldmodule, image_filenames):
+    """
+    Create an image field using the given fieldmodule.  The image filename must exist and
+    be a known image type.
+
+    :param fieldmodule: The fieldmodule to create the field in.
+    :param image_filename: Image filename.
+    :return: The image field created.
+    """
+    image_field = fieldmodule.createFieldImage()
+    image_field.setName('image_field')
```

Comment:
Suggest making this an optional argument to function, e.g. fieldname = 'image_field'.